microsoft/testsuites/asr/cvt.py

```diff
@@ -0,0 +1,305 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Move the files into vm_extensions folder

Reply:
CVT test validates the functionality of ASR driver.
The driver is installed on the system using extension.
The test is performed post installing the extensions